src/routes/+layout.svelte

```diff
@@ -1,5 +1,6 @@
-<script lang="ts">
+<script type="module" lang="ts">
```

Comment:
```suggestion
<script lang="ts">
```

setcontext doesn't work inside the module script